app/components/verenigingsregister/out-of-date-message.gjs

```diff
@@ -1,18 +1,74 @@
 import AuAlert from '@appuniversum/ember-appuniversum/components/au-alert';
+import AuLoader from '@appuniversum/ember-appuniversum/components/au-loader';
+import AuTooltip from '@appuniversum/ember-appuniversum/components/au-tooltip';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
 
-<template>
-  <AuAlert
-    @skin="warning"
-    @icon="alert-triangle"
-    @size="small"
-    @closable={{true}}
-    class="au-u-max-width-small"
-    ...attributes
-  >
-    Na een aanpassing kan het even duren voor de veranderingen zichtbaar zijn in
-    het Verenigingsloket Lokale Besturen.
-    <br />
-    Wees gerust, de veranderingen zijn vrijwel meteen zichtbaar in het
-    Verenigingenregister.
-  </AuAlert>
-</template>
+const POLLING_DELAY = 10_000;
+
+export default class OutOfDateMessage extends Component {
+  @service store;
+  intervalId;
+  currentEtag;
+
+  constructor() {
+    super(...arguments);
+
+    this.currentEtag = this.args.association.etag;
+    // this.pollForUpdates();
+  }
+
+  pollForUpdates() {
+    this.intervalId = setInterval(async () => {
+      await this.args.association.reload();
+
+      const newEtag = this.args.association.etag;
+      if (newEtag !== this.currentEtag) {
+        this.args.onUpdateAvailable?.();
+      }
+    }, POLLING_DELAY);
+  }
+
+  <template>
+    <AuAlert
+      @skin="warning"
+      @icon="alert-triangle"
+      @size="small"
+      @closable={{true}}
+      class="au-u-max-width-small"
+      ...attributes
+    >
+      Na een aanpassing kan het even duren voor de veranderingen zichtbaar zijn
+      in het Verenigingsloket Lokale Besturen.
+      <br />
+      Wees gerust, de veranderingen zijn vrijwel meteen zichtbaar in het
+      Verenigingenregister.
+
+      <AuTooltip @placement="bottom" as |tooltip|>
+        {{! This wrapper div is needed to position the tooltip }}
+        <div class="au-u-flex">
+          <AuLoader
+            @inline={{true}}
+            @centered={{false}}
+            @hideMessage={{true}}
```

Comment:
Don't you want to show the text? You added a paragraph below that is now hidden. I think the text adds value, also because not everyone is going to hover over the loader.